libraries/core-react/src/Breadcrumbs/Breadcrumb.tsx

```diff
@@ -1,12 +1,12 @@
-// @ts-nocheck
 import React, { forwardRef } from 'react'
-import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
 import { Typography } from '../Typography'
 import { Tooltip } from '../Tooltip'
 import { breadcrumbs as tokens } from './Breadcrumbs.tokens'
 
-const StyleTypography = styled(Typography)`
+type StyledProps = Pick<Props, 'maxWidth'>
```

Comment:
`Pick` is one of those things I didn’t know about, that we should probably use in more components…

Reply:
Recommend having a look at the Typescript [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html)  :)

Reply:
It's pretty neat! 😄  @mimarz also taught me `Omit<Type, Keys>` from [Typescript Utility Types](https://www.typescriptlang.org/docs/handbook/utility-types.html) where you can remove `Keys` from `Type` and re-use it